packages/create-block/lib/index.js

```diff
@@ -3,72 +3,68 @@
  */
 const program = require( 'commander' );
 const inquirer = require( 'inquirer' );
+const { startCase } = require( 'lodash' );
 
 /**
  * Internal dependencies
  */
 const CLIError = require( './cli-error' );
-const {
-	error,
-	info,
-} = require( './log' );
+const log = require( './log' );
 const { version } = require( '../package.json' );
 const scaffold = require( './scaffold' );
-const {
-	getDefaultAnswers,
-	getPrompts,
-} = require( './templates' );
-const { startCase } = require( './utils' );
+const { getDefaultValues, getPrompts } = require( './templates' );
 
 const commandName = `wp-create-block`;
 program
 	.name( commandName )
 	.description(
 		'Generates PHP, JS and CSS code for registering a block for a WordPress plugin.\n\n' +
-		'[slug] is optional. When provided it triggers the quick mode where it is used ' +
-		'as the block slug used for its identification, the output location for scaffolded files, ' +
-		'and the name of the WordPress plugin. The rest of the configuration is set to all default values.'
+			'[slug] is optional. When provided it triggers the quick mode where it is used ' +
+			'as the block slug used for its identification, the output location for scaffolded files, ' +
+			'and the name of the WordPress plugin. The rest of the configuration is set to all default values.'
```

Comment:
I don't necessarily see this as necessary (or correct for that matter)? Seemed fine (correct) before. Not a big deal either way.

Reply:
I executed Prettier on the package out of curiosity, the number of changes applied  was very small 😃this is one of them